README.md

```diff
@@ -34,6 +34,8 @@ npm i amqp-replay -g
 AMQP_URI="amqp://user:password@rabbitmq.ndd.com:5672/%2F" AMQP_QUEUE_NAME="social.test" AMQP_EXCHANGE_NAME="fgtest" amqp-replay
 ```
 
+You can add `INFINITE=true` for an infinite replay of the queue.
```

Comment:
I think that, most of the time, you only want to replay dead lettered messages. So false by default is the most logical IMHO :)